js/src/Ice/Stream.js

```diff
@@ -2289,14 +2289,14 @@ class WriteEncaps {
 
 export class OutputStream {
     constructor(arg1, arg2) {
-        this._instance = null;
+        var instance = null;
```

Comment:
I didn't change the logic of this constructor.

We should consider using the MATLAB style, with an optional encoding followed by an optional format + provide a way to retrieve encoding/format from the communicator.